.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,32 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '0 6 * * 1'
```

Comment:
Jeg ser at https://github.com/oslokommune/okctl kjører CodeQL for hver PR. Vi får se an hvor ofte vi ønsker å kjøre det etterhvert.

Reply:
Never mind, jeg som er blind 😅

Reply:
😁